addons/payment/views/payment_templates.xml

```diff
@@ -22,7 +22,7 @@
             - 'verify_validity' if we need to verify if the payment method is valid when adding a new one
             - 'prepare_tx_url' the url of the route which will handle the creation of a transaction for a form base payment (handles if the transaction is form or form_save)
         -->
-        <form t-if="s2s_acquirers or pms or form_acquirers" method="post" class="o_payment_form mt-3 clearfix"
+        <form t-if="pms or form_acquirers" method="post" class="o_payment_form mt-3 clearfix"
```

Comment:
Since you are merging the variables s2s_acquirers & form_acquirers, you should rename form_acquirers to "acquirers".

You should also update the "docstring" right above :)